benchopt/plotting/plot_bar_chart.py

```diff
@@ -38,6 +42,8 @@ def plot_bar_chart(df, obj_col='objective_value', plotly=False):
     fig = get_figure(plotly)
     c_star = df[obj_col].min() + eps
     for i, solver_name in enumerate(solver_names):
+        if reset_style and i == 0:
+            reset_solvers_idx()
```

Comment:
Maybe better to call before the loop no?

Reply:
It would have been better indeed, but anyway I removed those lines as I removed the `reset_style` argument from `plot_benchmark()`